examples/java/io/mailtrap/examples/contactexports/ContactExportsExample.java

```diff
@@ -0,0 +1,39 @@
+package io.mailtrap.examples.contactexports;
+
+import io.mailtrap.config.MailtrapConfig;
+import io.mailtrap.factory.MailtrapClientFactory;
+import io.mailtrap.model.request.contactexports.ContactExportFilter;
+import io.mailtrap.model.request.contactexports.ContactExportFilterOperator;
+import io.mailtrap.model.request.contactexports.CreateContactsExportRequest;
+
+import java.util.List;
+
+public class ContactExportsExample {
+
+    private static final String TOKEN = "<YOUR MAILTRAP TOKEN>";
+    private static final long ACCOUNT_ID = 1L;
+    private static final long CONTACT_LIST_ID = 1L;
+
+    public static void main(String[] args) {
+        final var config = new MailtrapConfig.Builder()
+            .token(TOKEN)
+            .build();
+
+        final var client = MailtrapClientFactory.createMailtrapClient(config);
+
+        final var exportFilter = ContactExportFilter.ids(ContactExportFilterOperator.EQUAL, CONTACT_LIST_ID);
```

Comment:
This example is outdated now


```suggestion
        final var exportFilter = ContactExportFilter.listIDs(ContactExportFilterOperator.EQUAL, CONTACT_LIST_ID);
```